elixir/apps/domain/test/support/new_fixtures/account_fixtures.ex

```diff
@@ -10,7 +10,7 @@ defmodule Domain.AccountFixtures do
   Generate valid account attributes with sensible defaults.
   """
   def valid_account_attrs(attrs \\ %{}) do
-    unique_num = System.unique_integer([:positive, :monotonic])
+    unique_num = System.unique_integer([:positive])
```

Comment:
Nothing wrong with this necessarily, just wondering if I need to start removing the `:monotonic` attribute in all of the other fixtures?